0008-atomic-registry.md

```diff
@@ -0,0 +1,144 @@
+
+### User Authentication
+
+Katello uses either its own builtin authentication or LDAP ([Foreman LDAP Authentication](https://theforeman.org/manuals/1.12/index.html#4.1.1LDAPAuthentication)). Atomic Registry leverages openshift/origin authentication and is able to run in several modes, including LDAP ([Configuring Authentication and User Agent](http://docs.projectatomic.io/registry/latest/install_config/configuring_authentication.html)). Using LDAP would allow a closer feel to SSO but only because the user names would be consistent. Each tool would need those users configured individually.
+
+### Role Based Access Control (RBAC)
+
+
+Katello has a rich, fine-grained RBAC model ([Roles and Permissions](https://theforeman.org/manuals/1.12/index.html#4.1.2RolesandPermissions)) that ties resources to actions. For example, a permission to view and sync products could be assigned to a user to allow them to bring in updated content but that would prevent the creation or deletion of products. Further refinement is possible by adding a search filter.
+
+The following use cases will be necessary:
+ * As a Katello admin, I wish to limit which users may update all docker repositories in a product.
+ * As a Katello admin, I wish to limit which users may update specific repositories in a product.
+ * As a Katello admin, I wish to limit which users may publish a new version of a contentview.
+ * As a Katello admin, I wish to limit which users may promote a contentview to a lifecycle environment.
+ * As a Katello admin, I wish to put restrictions on who may sync new tags to Atomic Registry.
+ * As a Katello admin, I wish to limit which uesrs may edit the settings on a docker repository.
+
+# Installation
+[installation]: #installation
+
+### All-in-one Server
+
+```shell
+# hostname -f
+katello.example.com
+
+# ...katello install instructions...
+  ... configures Katello registry at katello.example.com:5001
+  ... configures Atomic Registry at katello.example.com:5000
+  ... configures Atomic Registry UI at katello.example.com:9091
+
+# yum install -y atomic
+...
+Installed:
+  atomic.x86_64 1:1.10.5-7.el7 
+  
+# atomic install projectatomic/atomic-registry-install katello.example.com
+Using default tag: latest
+Trying to pull repository docker.io/projectatomic/atomic-registry-install ...
+...
+
+# systemctl enable --now atomic-registry-master.service
+# docker ps
+CONTAINER ID        IMAGE                COMMAND
+f58273e3fedc        cockpit/kubernetes   "/usr/libexec/cockpit"
+8cdb51b35239        openshift/origin    "/usr/bin/openshift s"
+
+# /var/run/setup-atomic-registry.sh katello.example.com
+Launch web console in browser at https://katello.example.com:9090
+By default, ANY username and ANY password will successfully authenticate.
+
+```
```

Comment:
What are these steps doing that the "...configures" steps on lines 111 - 113 are not doing?